.github/workflows/sast.yml

```diff
@@ -0,0 +1,17 @@
+name: SAST - Static application security testing
+
+on:
+  workflow_call:
+  workflow_dispatch:
+  push: # Until added to release pipeline
+
+jobs:
+  appsec-workflow:
+    uses: ZupIT/appsec-workflow/.github/workflows/appsec-workflow.yml@v1.3
+    with:
+      debug: false
+      repository: ${{github.event.repository.name}}
+      organization: ${{github.repository_owner}}
+      environment: main
+    secrets:
+      report-secret: ${{ secrets.VULN_MNG_REPORT_KEY }}
```

Comment:
This secret does not exist in this repo.

Reply:
Let's wait the APP SEC team to add it before merging this PR.